lib/internal/process/policy.js

```diff
@@ -57,5 +61,24 @@ module.exports = ObjectFreeze({
 
   assertIntegrity(moduleURL, content) {
     this.manifest.assertIntegrity(moduleURL, content);
+  },
+
+  deny(permissions) {
```

Comment:
I think we should throw an error if `permissions` does not match an existing permission. A typo here would be bad for security.